README.md

```diff
@@ -1,2 +1,10 @@
 # Software
 The Embedded Software Repo
+
+
+## System Requirements
+
+1. The PowerPlant shall be capable of operating for greater than 24 hours on a single full charge of its battery
+2. The PowerPlant shall stabilize its location to within 50 cm of the local maximum of light intensity
```

Comment:
Recommendation is to change this to 10 cm

Reply:
Good catch. 50 cm would have been way too big. Checkout commit: 5e97aff6a112cc7